src/ui/PageButton.tsx

```diff
@@ -8,21 +9,27 @@ export type PageButtonProps = {
 
 function PageButton(props: React.PropsWithoutRef<PageButtonProps>) {
   const { children, onClick, ...rest } = props;
+  const mainBgColor = useColorModeValue('ui.white', 'ui.black', 'ui.sepia');
```

Comment:
Maybe in black let's make it `gray.700` or something and in sepia mode `white`? Idk what the old version does but that seems good to me

Reply:
This style is for the Button wrapper, which I think should match the color of the Iframe. Perhaps you mean to change the color of the Circle?

Reply:
Oh true yes